### DISCUSSION TITLE
Is it possible with react-native to create my graphical environment in linux?

### DISCUSSION THREAD

**Discussion by ghost** [react-native-community/discussions-and-proposals#507] [Ideas]
Hi everyone!

I would like to know if it is possible to create a linux graphical environment with react-native. So my question would be... in the context... of react-native's current capabilities, would this be possible? Could I create my graphical environment on linux like something like XFCE, Gnome etc?

*The reason for this question is that... I, for example, build a lot of websites with react. Also, I build a lot of apps on android/ios with react-native. I really like the react-native and linux project, I would like to use these two things - for example creating my GUI in linux with react-native.*

**Comment by cortinico** [comment]
I would suggest you look into React Native Skia as that's probably the way to go if you want to render on Linux: https://github.com/shopify/react-native-skia

**Reply by Kudo** [reply]
or maybe https://github.com/react-native-skia/react-native-skia. the project is still in POC. it is still lack of some features.
(the react-native-skia naming is little confusing i know 😅)

**Reply by ghost** [reply]
Hi cortinico and Kudo, thank you for feedback.

**Comment by Valerio-boi** [comment]
thx
